README.md

```diff
@@ -13,8 +13,8 @@ and can generate documentation in multiple formats including standard Javadoc, H
 _Note: If you are upgrading from 0.10.x to a current release of Dokka, please have a look at our 
 [migration guide](runners/gradle-plugin/MIGRATION.md)_
 
-The preferred way is to use `plugins` block. Since Dokka is currently not published to the Gradle plugin portal, 
-you not only need to add `org.jetbrains.dokka` to the `build.gradle.kts` file, but you also need to modify the `settings.gradle.kts` file: 
+The preferred way is to use the `plugins` block. Since Dokka is currently not published to the Gradle plugin portal, 
+along with adding `org.jetbrains.dokka` to the `build.gradle.kts` file, and you **also** need to modify the `settings.gradle.kts` file: 
```

Comment:
I don't understand this change

I get that previous version was not perfect since `you` was duplicated but imho it was simpler to understand.